### ISSUE TITLE
typo in patch_model.py 

### ISSUE THREAD

**Issue by EricDeveaud** [kelwyres/Bactabolize#19]
Hello while running the examples I had the following error whith patching model onne: https://github.com/kelwyres/Bactabolize/blob/main/data/test_data/README.md#patch_model-for-non-functioning-draft-models.

see:

```
bactabolize patch_model --draft_model_fp data/model_patching_example/KPN2285_model.json --ref_model_fp KpSC_pan-metabolic_model_v1.json --patch_fp KPN2285_patch.json --biomass_reaction_id BIOMASS_Core_Oct2019 --output_fp KPN2285_patched.json

========================================
Traceback (most recent call last):
  File "/opt/gensoft/exe/Bactabolize/1.0.0/bin/bactabolize", line 8, in <module>
    sys.exit(entry())
  File "/opt/gensoft/exe/Bactabolize/1.0.0/venv/lib/python3.8/site-packages/bactabolize/__main__.py", line 26, in entry
    patch_model.run(config)
  File "/opt/gensoft/exe/Bactabolize/1.0.0/venv/lib/python3.8/site-packages/bactabolize/patch_model.py", line 15, in run
    print('Patching model ' + config.model_fp.stem)
AttributeError: 'ConfigPatchModel' object has no attribute 'model_fp'
```

my guess is that it should be `print('Patching model ' + config.ref_model_fp.stem)` instead.

**Comment by bananabenana** [comment]
Hi @EricDeveaud , thanks for bringing this to our attention. 

This should be fixed in version 1.0.1. - see commit https://github.com/kelwyres/Bactabolize/commit/e0fb9edeebd8178f681911b5a0942bcce2241982

The relevant line 15 required changing from 

```print('Patching model ' + config.model_fp.stem)```

to

```print('Patching model ' + config.draft_model_fp.stem)```

Tested and working.

Please reinstall with 
```
mamba create -n bactabolize -y -c scwatts -c bioconda -c conda-forge 'bactabolize==1.0.1'
```

**Comment by EricDeveaud** [comment]
maybee it will worth the generation of a release tag

regards

Eric
